app/mmstyle.h

```diff
@@ -178,6 +181,12 @@ class MMStyle: public QObject
      * Add new sizes here only if they need to be used in multiple files
      */
 
+    // Icon sizes
+    Q_PROPERTY( double icon16 READ icon16 CONSTANT )
+    Q_PROPERTY( double icon24 READ icon24 CONSTANT )
+    Q_PROPERTY( double icon32 READ icon32 CONSTANT )
+    Q_PROPERTY( double icon40 READ icon40 CONSTANT )
```

Comment:
Based on yesteday talk with Andrej I had impression we do not have 40px icons, do you see one somewhere? 40px is not icon but image or button. I would remove it from icon sizes here and also from gallery